pp/entrypoint/series_data_data_storage.cpp

```diff
@@ -142,6 +143,36 @@ extern "C" void prompp_series_data_data_storage_query(void* args, void* res) {
   }
 }
 
+extern "C" void prompp_series_data_data_storage_query_new(void* args, void* res) {
+  using Query = series_data::querier::Query<Slice<LabelSetID>>;
+  using entrypoint::series_data::RangeQuerierWithArgumentsWrapperNew;
+  using series_data::querier::Querier;
+
+  struct Arguments {
+    DataStoragePtr data_storage;
+    Query query;
+  };
+
+  struct Result {
+    QuerierVariantPtr querier{};
+    QueryStatus status;
+    entrypoint::head::SerializedDataPtr serialized_data{};
```

Comment:
```suggestion
    entrypoint::head::SerializedDataPtr *serialized_data{};
```
You need to receive pointer to unique_ptr